Source/Core/DolphinWX/NetPlay/NetPlaySetupFrame.cpp

```diff
@@ -304,135 +282,53 @@ NetPlaySetupFrame::~NetPlaySetupFrame()
   main_frame->g_NetPlaySetupDiag = nullptr;
 }
 
-void NetPlaySetupFrame::MakeNetPlayDiag(int port, const std::string& game, bool is_hosting)
-{
-  NetPlayDialog*& npd = NetPlayDialog::GetInstance();
-  NetPlayClient*& netplay_client = NetPlayDialog::GetNetPlayClient();
-
-  bool trav = !is_hosting && m_direct_traversal->GetCurrentSelection() == TRAVERSAL_CHOICE;
-
-  std::string ip;
-  npd = new NetPlayDialog(m_parent, m_game_list, game, is_hosting);
-  if (is_hosting)
-    ip = "127.0.0.1";
-  else if (trav)
-    ip = WxStrToStr(m_connect_hashcode_text->GetValue());
-  else
-    ip = WxStrToStr(m_connect_ip_text->GetValue());
-
-  IniFile inifile;
-  inifile.Load(File::GetUserPath(F_DOLPHINCONFIG_IDX));
-  IniFile::Section& netplay_section = *inifile.GetOrCreateSection("NetPlay");
-
-  std::string centralPortString = GetTraversalPort(netplay_section);
-  unsigned long int centralPort;
-  StrToWxStr(centralPortString).ToULong(&centralPort);
-
-  std::string centralServer = GetTraversalServer(netplay_section);
-
-  netplay_client = new NetPlayClient(ip, (u16)port, npd, WxStrToStr(m_nickname_text->GetValue()),
-                                     trav, centralServer, (u16)centralPort);
-  if (netplay_client->IsConnected())
-  {
-    int winPosX, winPosY, winWidth, winHeight;
```

Comment:
The original PR had removed `MakeNetPlayDiag`. `MakeNetPlayDiag` was only called from `NetPlaySetupFrame::DoHost` (not `NetPlayLauncher::Join`). Should I reintroduce the window saving only for `DoHost`?